### PR TITLE
Add queries for config runtime, task runtime, and test case runtime

### PR DESCRIPTION
**Pull request by jmaher**

### CODE REVIEW

**Comment by jmaher** [comment]
@klahnakoski - can you sanity check the .query files in here
@ahal - can you sanity check the python in here

of course if either of you see anything to change, just ask!

File: adr/queries/test_durations.query [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+limit: 10000
+where:
+    and:
+        - in: {repo.branch.name: {$eval: branch}}
+        - gte: [repo.push.date, {date: {$eval: from_date}}]
+        - lte: [repo.push.date, {date: {$eval: to_date}}]
+        - eq: {build.type: {$eval: config}}
+        - eq: {run.machine.platform: {$eval: platform}}
+        - regex: {result.test: ".*/.*"}
+        - neq: {result.ok: "false"}
```
Comment (klahnakoski) [inline adr/queries/test_durations.query]: Use {"eq":{"result.ok":"T"}} - booleans are stored as 'T' and 'F' in ES.

File: adr/recipes/task_durations.py [hunk 2]
```diff
@@ -9,10 +9,14 @@
...
 def run(args):
     parser = RecipeParser('date')
-    parser.add_argument('-b', '--branch', default=['autoland', 'mozilla-inbound', 'mozilla-central'],
+    parser.add_argument('-b', '--branch', default=['mozilla-central'],
                         help="Branches to gather backout rate on, can be specified "
                              "multiple times.")
-    parser.add_argument('--limit', type=int, default=50,
+    parser.add_argument('-p', '--platform', default='windows10-64',
+                        help="platform for results, default is windows10-64")
+    parser.add_argument('-c', '--config', default='opt',
```
Comment (ahal) [inline adr/recipes/task_durations.py]: I think this should be called 'build_type'

File: adr/recipes/test_durations.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+from ..query import format_date, run_query
+
+
+def run(args):
+    parser = RecipeParser('date')
+    parser.add_argument('-b', '--branch', default=['mozilla-inbound'],
+                        help="Branches to query results from.")
+    parser.add_argument('-p', '--platform', default='windows10-64',
+                        help="platform for results, default is windows10-64")
+    parser.add_argument('-c', '--config', default='opt',
```
Comment (ahal) [inline adr/recipes/test_durations.py]: I think this should be called 'build_type'

**Pull request merged**